### PR TITLE
Update event_handler.py

### PR DESCRIPTION
**Pull request by Timmie644**
for SMP_ping(don't know if it will work)

### CODE REVIEW

File: cogs/event_handler.py [hunk 1]
```diff
@@ -28,6 +30,10 @@ class event_handler(
     def __init__(self, bot: commands.Bot):
         self.bot = bot
         self.bot.add_view(hypixel_ping_buttons())
+    
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.bot.add_view(SMP_ping_buttons())
```
Comment (codeman1o1) [inline cogs/event_handler.py:36]: Remove the duplicate __init__ function

File: cogs/event_handler.py [hunk 2]
```diff
@@ -100,6 +106,33 @@ async def on_message(self, message):
                     delete_after=600,
                     view=hypixel_ping_buttons(),
                 )
+         @commands.Cog.listener()
+    async def on_message(self, message):
```
Comment (codeman1o1) [inline cogs/event_handler.py:110]: Remove the duplicate on_message function

Comment (Timmie644) [reply cogs/event_handler.py:110]: ok tnx

**Review by codeman1o1 — CHANGES_REQUESTED** [review]
Remove duplicate functions

**Review by codeman1o1 — APPROVED** [review]

**Pull request merged**